src/ContainerResolver.php

```diff
@@ -61,10 +65,16 @@ private static function resolveDefaultContainer(): ContainerInterface
 
     private static function resolveMvcContainer(): ContainerInterface
     {
+        /**
+         * @psalm-suppress MissingFile
+         * @psalm-var array<string, mixed> $appConfig
+         */
         $appConfig = include 'config/application.config.php';
         if (file_exists('config/development.config.php')) {
+            /** @psalm-var array<string, mixed> $appConfig */
             $appConfig = ArrayUtils::merge(
```

Comment:
I wonder if we should `Assert::isMap()` on both `$appConfig` and the development config :thinking: